pkg/redact/redact.go

```diff
@@ -48,6 +49,36 @@ func Redact(msg interface{}, val reflect.Value) {
 	}
 }
 
+// Redact - removes data from fields marked as sensitive given a proto message struct
```

Comment:
NIT: update comments to reflect right function name and intent.  Same for the below two functions.

Reply:
added/updated comments